spec/unit/middleware/service_broker_rate_limiter_spec.rb

```diff
@@ -60,27 +61,42 @@ module Middleware
           context 'when the path is /v2/*' do
             let(:path_info) { '/v2/service_instances' }
             it 'formats the response error in v2 format' do
-              _, _, body = middleware.call(user_env)
+              _, response_headers, body = middleware.call(user_env)
               json_body = JSON.parse(body.first)
               expect(json_body).to include(
                 'code' => 10016,
                 'description' => 'Service broker concurrent request limit exceeded',
                 'error_code' => 'CF-ServiceBrokerRateLimitExceeded',
               )
+              expect(response_headers['Retry-After']).to be_between(Time.now + broker_timeout * 0.5, Time.now + broker_timeout * 1.5)
```

Comment:
The [unit test run](https://github.com/cloudfoundry/cloud_controller_ng/actions/runs/2027306272) for the merged commit failed. I would suggest to use `Timecop` to freeze the time before issuing the call.

Reply:
Fixed with #2733